controls/attr_works.rb

```diff
@@ -0,0 +1,24 @@
+title 'Tests to confirm attr works as expected'
+
+plan_name = input('plan_name', value: 'attr')
+plan_ident = "#{ENV['HAB_ORIGIN']}/#{plan_name}"
+
+control 'core-plans-attr-works' do
+  impact 1.0
+  title 'Ensure attr works as expected'
+  desc '
+  '
+  attr_path = command("hab pkg path #{plan_ident}")
+  describe attr_path do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+  end
+  
+  attr_pkg_ident = ((attr_path.stdout.strip).match /(?<=pkgs\/)(.*)/)[1]
+  describe command("DEBUG=true; hab pkg exec #{ attr_pkg_ident} getfattr --version") do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
+    its('stdout') { should match /getfattr 2.4.48/ }
```

Comment:
Would it be worthwhile doing something similar to the bats test here instead of hard-coding the version string maybe?
```
TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
```
I think if you do something like this it should work e.g. 
```
its('stdout') { should match /getfattr #{attr_pkg_ident.split('/')[2] }
```

Reply:
Thanks @skpaterson.  I'll make the change and have also raised an issue https://github.com/chef-base-plans/base-plan-skeleton/issues/3 as discussed.